evalml/preprocessing/utils.py

```diff
@@ -62,6 +62,31 @@ def split_data(
 
     Returns:
         pd.DataFrame, pd.DataFrame, pd.Series, pd.Series: Feature and target data each split into train and test sets.
+
+    Examples:
+        >>> X = pd.DataFrame([1, 2, 3, 4, 5, 6], columns=["First"])
+        >>> y = pd.Series([8, 9, 10, 11, 12, 13])
+        >>> splits = split_data(X, y, "regression", random_seed=42)
```

Comment:
Might be clearer if we unpack the tuple as `X_train, X_validation, y_train, y_validation = split_data(...)`